modules/oxide/src/less/theme/components/well/well.less

```diff
@@ -33,6 +33,11 @@
     border-radius: @well-border-radius;
     display: flex;
     flex: 1;
+    overflow: hidden;
     position: relative;
+
+    &:focus-within {
```

Comment:
Maybe we need a changelog item for this like that we are adding focusing to custom editors. If a customer has a custom editor that will no we focused as well. Very unlikely but still a change in behavior to the better.

Reply:
added